tickets/src/components/content/EventContent.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { Fieldset, Field, Label } from './CreateContent'
+import { MONTH_NAMES } from '../../constants'
+import UnlockPropTypes from '../../propTypes'
+import BalanceProvider from '../helpers/BalanceProvider'
+
+export const EventContent = ({ name, description, location, date, lock }) => {
+  let dateString =
+    MONTH_NAMES[date.getMonth()] +
+    ' ' +
+    date.getDate() +
+    ', ' +
+    date.getFullYear()
+
+  return (
+    <>
+      <Title>{name}</Title>
+      <DetailsFieldset>
+        <Field>
+          <Label>Ticket Price</Label>
+          <BalanceProvider
```

Comment:
Any idea why this does not seem to work on the screenshots you sent?

Reply:
None, but I'm going to come back to it in a follow-up.